src/widget/container/alignment.rs

```diff
@@ -0,0 +1,12 @@
+#[derive(Clone, Copy, Eq, PartialEq)]
+pub enum Alignment {
+    Start,
+    Center,
+    End,
+}
+
+impl Default for Alignment {
+    fn default() -> Self {
+        Self::Start
```

Comment:
For comparison, since `alignment` seems to correspond to flexbox `align-items`, there the default is `stretch`

Reply:
Yeah, I will probably switch to that eventually, but the cross-axis sizing behavior isn’t really thought through yet.